conf/lambda.json

```diff
@@ -23,6 +23,18 @@
       "subnet_ids": []
     }
   },
+  "athena_partition_refresh_config": {
```

Comment:
is this missing the `results_bucket`?

Reply:
also `database_name`?

Reply:
those options are only needed if the user wants to override the defaults that we use. I can add the defaults here but have some concerns. Mainly, I worry that our config is growing unnecessarily large with superfluous options that the vast majority of users wouldn't need to worry about. What are your thoughts? Have it here or omit it and just document the options config settings well?